includes/abstracts/abstract-wc-order.php

```diff
@@ -344,7 +344,7 @@ public function get_status( $context = 'view' ) {
 	 * Get discount_total.
 	 *
 	 * @param  string $context View or edit context.
-	 * @return string
+	 * @return float
```

Comment:
We should stay using `string` for this method and for all other methods.

See:

```bash
wp> $order->get_discount_total()
=> string(1) "10.00"
```

Reply:
Thank you!